### ISSUE TITLE
Compiling failed: no such file or directory

### ISSUE THREAD

**Issue by Ewebalvin** [ephread/Instructions#89]
```
pod 'Instructions', '~> 0.5'
pod install
```

Here was the error message:
`no such file or directory: '/Pods/Instructions/Source/CoachMarkPosition.swift'`

I checked the folders in Pods. There were only 2 sub folders, not including "Source".
```
Pods
  |-- Instructions
      |--Core
      |--Support Files
```

**Comment by ephread** [comment]
Hello @Ewebalvin, that's awkward. The `Sources` directory is included in the Podspec for 0.5.0, I'll double check on my end, but it might related to CocoaPods caching issues.

**Comment by ephread** [comment]
Instructions has been updated to 1.0.0-beta.1. I'm assuming this solves the issue, if not, feel free to re-open the issue.
